core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/devfile/Devfile.java

```diff
@@ -46,4 +45,7 @@
 
   /** Returns devfile attributes. Devfile attributes must not contain null keys or values. */
   Map<String, String> getAttributes();
+
+  /** @return the metadata of the devfile. */
```

Comment:
To use one style, please consider rewriting it like the following: `Returns ...`